chapters/overview.md

```diff
@@ -93,6 +93,10 @@ prior knowledge of AllenNLP—the whole point of this guide is to provide onboar
 
 ## Part 3
 
+* [Semantic parsing: Intro and Seq2Seq model](/semantic-parsing-seq2seq)
+* [Semantic parsing: Grammar-based decoding](/semantic-parsing-grammar)
```

Comment:
Some of these should be labeled as "Coming soon", right?

Reply:
Yes. I indicated that these chapters are coming soon in the "Coming Soon" chapter. :)